internal/credentials/credentialstest/credentialstest.go

```diff
@@ -0,0 +1,42 @@
+package credentialstest
```

Comment:
Shouldn't this file be named `credentials_test.go` to match our pattern (and the unit test greps)?

Reply:
The naming for the package fits. It is named similarly to others like `loggingtest`: https://github.com/posit-dev/publisher/blob/7830e62c2bfe119d8d47c3dcc9dce19bacef5371/internal/logging/loggingtest/mock_logger.go#L1

The file naming looks to be something like `mock_something.go` though. I can update that to be more similar to other bits of the codebase.